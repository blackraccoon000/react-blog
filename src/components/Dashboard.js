import React from "react"
import NoteList from "./NoteList"
import NoteListFilters from "./NoteListFilters"

const Dashboard = (props) => {
  const history = props.history
  return (
    <div>
      <div className="content-container">
        {/* <h1>This Component is Dashboard</h1> */}
        <NoteListFilters/>
        <NoteList/>
        <button onClick={()=>{
          history.push("/")
        }}>Go Home</button>
      </div>
    </div>
  )
}

export default Dashboard