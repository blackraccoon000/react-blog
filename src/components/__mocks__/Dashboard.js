import React from "react"
const mock = jest.fn().mockImplementation(() => (<div>DashboardPage Use ManualMock</div>))
export default mock