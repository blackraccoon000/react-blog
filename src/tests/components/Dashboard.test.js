import React from "react"
import { shallow } from "enzyme"
import Dashboard from "../../components/Dashboard"

let wrapper

beforeEach(()=>{
  wrapper = shallow(<Dashboard/>)
})

it("should render Dashboard components",() => {
  expect(wrapper).toMatchSnapshot()
})