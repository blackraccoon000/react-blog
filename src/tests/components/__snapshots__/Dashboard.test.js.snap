// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dashboard components 1`] = `
<div>
  <div
    className="content-container"
  >
    <Connect(NoteListFilters) />
    <Connect(NoteList) />
    <button
      onClick={[Function]}
    >
      Go Home
    </button>
  </div>
</div>
`;
